src/components/Pools/Swap/columns.tsx

```diff
@@ -48,7 +48,7 @@ export const coverageColumn: ColumnDef<SwapPoolColumn> = {
 export const aprColumn: ColumnDef<SwapPoolColumn> = {
   header: 'APR',
   accessorKey: 'apr',
-  accessorFn: (row) => row.apr?.toString(),
+  accessorFn: (row) => row.reserves?.toString(), // TODO: get apr
```

Comment:
Is this something that we calculate or that the indexer needs to provide?

Reply:
This is something we did not resolve yet on nabla side either. We have a meeting tmr